README.md

```diff
@@ -107,17 +107,38 @@ Follow the guidelines in [CONTRIBUTING.md](https://github.com/CSE-25/quick_start
 
 ### Run the package.
 
-1. Run the following command in the terminal to start the package:
+To test the CLI tool locally, you need to link the package. Use a separate testing directory to avoid modifying files in the main quick_start_express directory.
+
+1. **Link the Package in the Main Directory**: In the quick_start_express directory, run:
 
    ```bash
-   npm link # in the quick_start_express directory
+   npm link
    ```
 
+2. **Create a Testing Directory**: Outside the quick_start_express directory, create a testing directory:
```

Comment:
Why not create a directory called `qse-test` within the root directory of `quick_start_express`. If we move out of scope, it might become hard to track files.